hpi/cloud/course/v1test/course.proto

```diff
@@ -81,6 +88,12 @@ message Course {
 
   // A website that contains more information about the course.
   string website = 6;
+
+  // An upper bound for the number of participants or null if there is none.
+  google.protobuf.UInt32Value attendance = 7;
+
+  // Required. The enrollment deadline.
+  google.type.Date enrollment_deadline = 8;
```

Comment:
You need to import these definitions (see for example https://github.com/HPI-de/hpi-cloud-apis/blob/dev/hpi/cloud/news/v1test/article.proto#L6).
- `google/protobuf/wrappers.proto` for `UInt32Value` (and all other `Value`s)
- `google/type/date.proto` for `Date` (I think)